### PR TITLE
blockdev_stream: support backend ceph

### PR DESCRIPTION
**Pull request by aliang123**
Add backend ceph support for block stream

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2046003

### CODE REVIEW

**Comment by aliang123** [comment]
(01/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_to_invalid_node.q35: PASS (55.31 s)
 (02/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_hotunplug.q35: PASS (111.21 s)
 (03/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_no_backing.q35: PASS (42.15 s)
 (04/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_forbidden_actions.q35: PASS (53.34 s)
 (05/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_powerdown.q35: PASS (140.82 s)
 (06/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_subchain.q35: PASS (210.12 s)
 (07/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_speed.q35: PASS (143.84 s)
 (08/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_general.auto_dismiss_on.auto_finalize_on.q35: PASS (186.16 s)
 (09/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_general.auto_dismiss_on.auto_finalize_off.q35: PASS (141.82 s)
 (10/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_general.auto_dismiss_off.auto_finalize_on.q35: PASS (147.92 s)
 (11/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_general.auto_dismiss_off.auto_finalize_off.q35: PASS (145.04 s)
 (12/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_multiple_blocks.q35: PASS (233.19 s)
 (13/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_filter_nodename.q35: PASS (68.34 s)
 (14/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_stress.q35: PASS (265.63 s)
 (15/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_none_existed_overlay.q35: PASS (39.00 s)
 (16/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_base_itself.q35: PASS (112.16 s)
 (17/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_vm_stop_cont.q35: PASS (237.56 s)
 (18/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_cor_base.q35: PASS (36.65 s)
 (19/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_on_error_ignore.q35: PASS (255.17 s)
 (20/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_vm_reboot.q35: PASS (250.23 s)
 (21/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_no_space.q35: PASS (164.73 s)
 (22/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream.q35: PASS (95.82 s)
 (23/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_install.q35: PASS (918.04 s)
 (24/24) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_cor.q35: PASS (119.90 s)

**Comment by aliang123** [comment]
Test cmd:
  python3 ConfigTest.py --guestname=RHEL.8.6.0 --platform=x86_64 --category=storage_vm_migration_blockdev --imageformat=raw --storage-backend-conf=/home/ceph.cfg --customsparams="no functional_live_snapshot functional_block_mirror functional_live_commit functional_incremental_live_backup"

**Comment by aliang123** [comment]
@zhencliu @ngu-niny @MiriamDeng, please help to review, thanks.

**Comment by MiriamDeng** [comment]
[root@ibm-p9wr-04 kar]# more /home/ceph.cfg
[ceph]
rbd_pool_name = rbd
Test command
python3 ConfigTest.py --guestname=RHEL.8.6.0 --platform=ppc64le --category=storage_vm_migration_blockdev --imageformat=raw --storage-backend-conf=/home/ceph.cfg --customsparams="no functional_live_snapshot functional_block_mirror functional_live_commit functional_incremental_live_backup" --debug

Test result,
...
/ (32/33) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_multiple_blocks: ERROR: 'CmdNotFoundError' object has no attribute 'message' (0.30 s)
 (33/33) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_to_invalid_node: STARTED
\ (33/33) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_to_invalid_node: ERROR: 'CmdNotFoundError' object has no attribute 'message' (0.27 s)
...

**Comment by MiriamDeng** [comment]
> [root@ibm-p9wr-04 kar]# more /home/ceph.cfg [ceph] rbd_pool_name = rbd Test command python3 ConfigTest.py --guestname=RHEL.8.6.0 --platform=ppc64le --category=storage_vm_migration_blockdev --imageformat=raw --storage-backend-conf=/home/ceph.cfg --customsparams="no functional_live_snapshot functional_block_mirror functional_live_commit functional_incremental_live_backup" --debug
> 
> Test result, ... / (32/33) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_multiple_blocks: ERROR: 'CmdNotFoundError' object has no attribute 'message' (0.30 s) (33/33) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_to_invalid_node: STARTED \ (33/33) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_to_invalid_node: ERROR: 'CmdNotFoundError' object has no attribute 'message' (0.27 s) ...
Set up the ceph server by the script, and test is going on

**Comment by MiriamDeng** [comment]
> 



> [root@ibm-p9wr-04 kar]# more /home/ceph.cfg [ceph] rbd_pool_name = rbd Test command python3 ConfigTest.py --guestname=RHEL.8.6.0 --platform=ppc64le --category=storage_vm_migration_blockdev --imageformat=raw --storage-backend-conf=/home/ceph.cfg --customsparams="no functional_live_snapshot functional_block_mirror functional_live_commit functional_incremental_live_backup" --debug
> 
> Test result, ... / (32/33) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_multiple_blocks: ERROR: 'CmdNotFoundError' object has no attribute 'message' (0.30 s) (33/33) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_to_invalid_node: STARTED \ (33/33) Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_to_invalid_node: ERROR: 'CmdNotFoundError' object has no attribute 'message' (0.27 s) ...

Setup the env again and re-run it, will update the results later.

**Comment by ngu-niny** [comment]
@aliang123 We do support rbd qcow2, so we don't need to limit the data image format as raw.

**Comment by ngu-niny** [comment]
> @aliang123 We do support rbd qcow2, so we don't need to limit the data image format as raw.

Talked with Aihua, since there is still product bug on rbd qcow2, will postpone to add it.

**Comment by MiriamDeng** [comment]
I ran it with raw format and most test cases passed excepted some test cases.
13	Host_RHEL.m8.u6.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_general.auto_dismiss_on.auto_finalize_off
17	Host_RHEL.m8.u6.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream	
21	Host_RHEL.m8.u6.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_subchain	
33	Host_RHEL.m8.u6.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_speed

**Comment by MiriamDeng** [comment]
The second one is as the following, it's related with ENV setup.
Start Time	UID	Test Name	Variant	Status	Time (sec)	Info	Debug
2022-02-14 08:09:12	16	Host_RHEL.m8.u6.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_stress		ERROR	89.40	Installation failed with output: checking for a BSD-compatible install... /usr/bin/install -c checking whether build environment is sane... configure: error: newly created file is older than distributed files! Check your system clock

**Comment by ngu-niny** [comment]
LGTM.

Most of the test case could pass, while the failed one should not be caused by the new ceph support:

022-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_vm_stop_cont Finshed 2022-02-14 12:24:53 2022-02-14 12:34:45 PASS 591.806992
023-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_base_itself Finshed 2022-02-14 12:34:45 2022-02-14 12:38:12 PASS 206.631756
024-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_subchain Finshed 2022-02-14 12:38:12 2022-02-14 12:41:58 PASS 226.464087
025-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_none_existed_overlay Finshed 2022-02-14 12:41:58 2022-02-14 12:43:32 PASS 93.590071
026-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream Finshed 2022-02-14 12:43:32 2022-02-14 12:46:48 FAIL 196.514646
027-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_general.auto_dismiss_on.auto_finalize_on Finshed 2022-02-14 12:46:49 2022-02-14 12:51:53 PASS 304.458753
028-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_general.auto_dismiss_on.auto_finalize_off Finshed 2022-02-14 12:51:53 2022-02-14 12:55:59 PASS 245.594249
029-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_general.auto_dismiss_off.auto_finalize_on Finshed 2022-02-14 12:55:59 2022-02-14 13:00:31 PASS 271.80256
030-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_general.auto_dismiss_off.auto_finalize_off Finshed 2022-02-14 13:00:31 2022-02-14 13:06:37 PASS 366.668207
031-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_install Finshed 2022-02-14 13:06:38 2022-02-14 13:26:38 PASS 1200.958556
032-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_on_error_ignore Finshed 2022-02-14 13:26:39 2022-02-14 13:30:15 PASS 216.460627
033-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_powerdown Finshed 2022-02-14 13:30:15 2022-02-14 13:33:34 FAIL 199.082713
034-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_hotunplug Finshed 2022-02-14 13:33:34 2022-02-14 13:37:01 PASS 206.274078
035-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_stress Finshed 2022-02-14 13:37:01 2022-02-14 13:38:52 ERROR 111.37772
036-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_cor_base Finshed 2022-02-14 13:38:52 2022-02-14 13:39:35 PASS 42.446194
037-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_forbidden_actions Finshed 2022-02-14 13:39:35 2022-02-14 13:41:47 PASS 132.44045
038-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_to_invalid_node Finshed 2022-02-14 13:41:47 2022-02-14 13:43:45 PASS 117.17174
039-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_multiple_blocks Finshed 2022-02-14 13:43:45 2022-02-14 13:48:31 PASS 285.856816
040-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_no_backing Finshed 2022-02-14 13:48:31 2022-02-14 13:50:04 PASS 93.599996
041-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_filter_nodename Finshed 2022-02-14 13:50:04 2022-02-14 13:52:13 PASS 128.58949
042-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_speed Finshed 2022-02-14 13:52:13 2022-02-14 13:56:36 FAIL 263.272839
043-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_no_space Finshed 2022-02-14 13:56:37 2022-02-14 13:59:28 PASS 171.717579
044-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_vm_reboot Finshed 2022-02-14 13:59:28 2022-02-14 14:09:31 PASS 602.691405
045-Host_RHEL.m8.u6.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_cor Finshed 2022-02-14 14:09:31 2022-02-14 14:13:07 PASS 215.512221

Acked-by: Nini Gu <ngu@redhat.com>

File: qemu/tests/cfg/blockdev_stream_dirty_bitmap.cfg [hunk 1]
```diff
@@ -29,3 +29,5 @@
         lun_data1 = 1
         enable_iscsi_data1sn = no
         image_raw_device_data1sn = no
+    ceph:
+        image_format_data1 = raw
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_stream_dirty_bitmap.cfg:33]: enable_ceph_data1sn = no

File: qemu/tests/cfg/blockdev_stream_filter_nodename.cfg [hunk 2]
```diff
@@ -30,3 +30,5 @@
         lun_data1 = 1
         enable_iscsi_sn1 = no
         image_raw_device_sn1 = no
+    ceph:
+        image_format_data1 = raw
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_stream_filter_nodename.cfg:34]: enable_ceph_sn1 = no

File: qemu/tests/cfg/blockdev_stream_on_error_ignore.cfg [hunk 3]
```diff
@@ -43,6 +43,9 @@
         remove_image_data1 = no
     iscsi_direct:
         lun_data1 = 1
+    ceph:
+        enable_ceph_data1 = yes
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_stream_on_error_ignore.cfg:47]: we can remove this line, it's enabled by default

**Comment by aliang123** [comment]
(1/3) Host_RHEL.m8.u6.product_rhel.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_dirty_bitmap.q35: PASS (45.81 s)                                      
(2/3) Host_RHEL.m8.u6.product_rhel.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_filter_nodename.q35: PASS (41.84 s)
(3/3) Host_RHEL.m8.u6.product_rhel.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.blockdev_stream_on_error_ignore.q35: PASS (60.47 s)

File: qemu/tests/cfg/blockdev_stream_filter_nodename.cfg (hunk 2 above)
Comment (aliang123) [reply qemu/tests/cfg/blockdev_stream_filter_nodename.cfg:34]: Have updated, thanks zhenchao.

File: qemu/tests/cfg/blockdev_stream_dirty_bitmap.cfg (hunk 1 above)
Comment (aliang123) [reply qemu/tests/cfg/blockdev_stream_dirty_bitmap.cfg:33]: Have updated, thanks zhenchao.

Comment (zhencliu) [reply qemu/tests/cfg/blockdev_stream_dirty_bitmap.cfg:33]: this one should be enable_ceph_data1sn instead of enable_ceph_sn1

**Comment by zhencliu** [comment]
LGTM

**Comment by MiriamDeng** [comment]
LGTM,thanks !
1/4) Host_RHEL.m8.u6.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_speed: PASS (249.20 s)
2/4) Host_RHEL.m8.u6.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream: PASS (210.97 s)
3/4) Host_RHEL.m8.u6.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_general.auto_dismiss_on.auto_finalize_off: PASS (288.18 s)
4/4) Host_RHEL.m8.u6.product_av.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_subchain: PASS (230.15 s)

**Pull request merged**